LabBuilder/LabBuilder.psm1

```diff
@@ -347,14 +347,14 @@ function Get-LabConfiguration {
     # Figure out the Config path and load it into the XML object (if we can)
     # This path is used to find any additional configuration files that might
     # be provided with config
-    [String] $ConfigPath = [System.IO.Path]::GetDirectoryName($Path)
+    [String] $ConfigPath = (Get-Location).Path
     [String] $XMLConfigPath = $Configuration.labbuilderconfig.settings.configpath
     if ($XMLConfigPath) {
         if ($XMLConfigPath.Substring(0,1) -eq '.')
```

Comment:
While working on another part of this code I found out the official .NET way of testing if a path is rooted or not:
```powershell
if (! [System.IO.Path]::::IsPathRooted($XMLConfigPath))
```

Do you want to change your branch with this method? If you make changes to your dev branch they will automatically get reflected in this PR without you doing anything.